todo-react/src/components/addTodo/style.css

```diff
@@ -0,0 +1,30 @@
+#bar {
```

Comment:
@Gayatri-innotech take class names proper. What is #bar ?? what is #ent??
Always create proper class names